toolkit/resources/imageconfigs/packagelists/iso-initrd-packages.json

```diff
@@ -34,27 +32,20 @@
         "grub2-pc",
         "gzip",
         "haveged",
-        "iputils",
-        "less",
         "libcap",
         "libgcc",
         "libstdc++",
-        "linux-firmware",
+        "linux-firmware-qlogic",
         "lvm2",
         "lua",
         "lz4",
         "ncurses",
         "ncurses-term",
-        "net-tools",
         "nspr",
         "nss",
-        "openssh-server",
         "openssl",
         "mariner-release",
-        "mariner-repos-shared",
```

Comment:
Do we need this `mariner-repos-shared` package for GPG key support in RPM/DNF?

Reply:
I would recommend to add a statement in the PR description to mention the next-step action items for supporting PXE boot scenarios so that other developers or customers won't get confused when they see that we're removing PXE-related components within this change.

Reply:
Our plan is to create or provide instructions to customers on how to build ISO for PXE boot scenarios. Mostly provide instructions on how to build PXE boot ISO until we finish our investigations to separate out rootfs filesystem to squashfs format.

Reply:
Added steps on how to build PXE boot needed initrd image with default ISO.